app/models/book_tour.rb

```diff
@@ -15,7 +15,8 @@ class BookTour < ApplicationRecord
   before_create :date_start
   validate :date_end
   delegate :title, to: :tour
-  scope :sort_tour, ->{order(created_at: :desc)}
+  delegate :name, to: :user
+  scope :sort_tour_deleted, ->{where(deleted: false).order(created_at: :desc)}
```

Comment:
a nghĩ là chỗ này nên improve như thế này:
- tạo scope `active` -> chỉ lọc ra book_tour chưa bị xóa.
- đổi tên scope `sort_tour_deleted` thành `newest`
-
```suggestion
  scope :newest, ->{active.order(created_at: :desc)}
```

Reply:
dạ để e sửa a